File: 프로그래머스/1/135808. 과일 장수/과일 장수.java
import java.util.*;
class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        Arrays.sort(score);
        int idx = score.length - m;
        while(idx >= 0) {
            answer += score[idx] * m;
            idx -= m;
                
        }
        return answer;
    }
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    var answer = 0;
    
    score.sort((a, b) => b - a);
    const apple = score.length; // 사과 개수
    const count = Math.floor(apple / m); // 상자 개수
    
   for (let i = 0; i < count; i++) {
       const apples = [];
       for (let j = i * m; j < i * m + m; j++) {
           apples.push(score[j]);
       }
       
       const min = Math.min(...apples);
      
       answer += min * m;
   }
    
   return answer;
}